CMakeLists.txt

```diff
@@ -224,13 +232,12 @@ MESSAGE("  Boost_LIBRARY_DIRS: ${Boost_LIBRARY_DIRS}")
 MESSAGE("  BOOST_LIBRARYDIR : ${BOOST_LIBRARYDIR}") 
 MESSAGE("  BOOST_INCLUDEDIR : ${BOOST_INCLUDEDIR}") 
 
-
 INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIRS})
 LINK_DIRECTORIES(${Boost_LIBRARY_DIRS})
 endif (WIN32)
 
 # Ensure that Boost is found. Environment variable BOOST_ROOT can help to find a user-specified path to Boost.
-SET(Boost_ADDITIONAL_VERSIONS "1.36.0" "1.37" "1.37.0" "1.38.0" "1.39.0" "1.40.0" )
+SET(Boost_ADDITIONAL_VERSIONS "1.59.0" "1.55.0" "1.36.0" "1.37" "1.37.0" "1.38.0" "1.39.0" "1.40.0" )
```

Comment:
I will leave that to you folks.   But for my testing I used 1.59 to get it to build.